frontend/test/metabase/pulse/pulse.unit.spec.js

```diff
@@ -1,57 +1,46 @@
 import React from "react";
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
 
 import RecipientPicker from "metabase/pulse/components/RecipientPicker";
-import TokenField from "metabase/components/TokenField";
-
-global.ga = jest.fn();
 
 const TEST_USERS = [
-  { id: 1, common_name: "Barb", email: "barb_holland@hawkins.mail" }, // w
-  { id: 2, common_name: "Dustin", email: "dustin_henderson@hawkinsav.club" }, // w
-  { id: 3, common_name: "El", email: "011@energy.gov" },
-  { id: 4, common_name: "Lucas", email: "lucas.sinclair@hawkins.mail" }, // w
```

Comment:
Any reason to change these? While these were originally just a fun Stranger Things joke, I do wonder if having different TLDs is a nice thing to keep around in case it were to expose something weird in a email regex or some such. (Although I realize that's not being tested here).

Reply:
@flamber argued many times before that we shouldn't use real TLDs. [`.test`](https://en.wikipedia.org/wiki/.test) is reserved TLD intended for exactly these kind of purposes. I started using it exclusively in all the tests ever since I first learned about reserved domains.

Reply:
Fair enough.